src/components/Receive.jsx

```diff
@@ -16,41 +15,46 @@ export default function Receive() {
   const location = useLocation()
   const settings = useSettings()
   const currentWallet = useCurrentWallet()
+  const walletInfo = useCurrentWalletInfo()
   const addressCopyFallbackInputRef = useRef()
   const [validated, setValidated] = useState(false)
   const [alert, setAlert] = useState(null)
   const [isLoading, setIsLoading] = useState(false)
   const [address, setAddress] = useState('')
   const [amount, setAmount] = useState('')
   const [account, setAccount] = useState(parseInt(location.state?.account, 10) || 0)
+  const [accounts, setAccounts] = useState([])
   const [addressCount, setAddressCount] = useState(0)
   const [showSettings, setShowSettings] = useState(false)
   const [addressCopiedFlag, setAddressCopiedFlag] = useState(0)
   const [showAddressCopiedConfirmation, setShowAddressCopiedConfirmation] = useState(false)
 
   useEffect(() => {
     const abortCtrl = new AbortController()
-    const fetchAddress = async (accountNr) => {
-      const { name: walletName, token } = currentWallet
-
-      setAlert(null)
-      setIsLoading(true)
-      Api.getAddressNew({ walletName, mixdepth: accountNr, token, signal: abortCtrl.signal })
-        .then((res) => (res.ok ? res.json() : Api.Helper.throwError(res, t('receive.error_loading_address_failed'))))
-        .then((data) => setAddress(data.address))
-        .catch((err) => {
-          !abortCtrl.signal.aborted && setAlert({ variant: 'danger', message: err.message })
-        })
-        .finally(() => !abortCtrl.signal.aborted && setIsLoading(false))
-    }
-
-    if (ACCOUNTS.includes(account)) {
-      fetchAddress(account)
-    }
```

Comment:
It's ok to get rid of that check since `accounts` can't be set manually anyway (it can just be selected from a pre-filled dropdown), right?

Reply:
Yes. And even if it could be set manually, it would only lead to an error being displayed, which feels better than just ignoring the request.

Reply:
Sounds good to me! ✅